plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ProgramConverter.kt

```diff
@@ -122,7 +122,14 @@ class ProgramConverter(val session: FirSession, override val config: PluginConfi
 
     override fun embedFunctionSignature(symbol: FirFunctionSymbol<*>): FunctionSignature {
         val retType = symbol.resolvedReturnTypeRef.type
-        val receiverType = symbol.receiverType
```

Comment:
`receiverType` is our own extension property, can this be handled there?  I think there is already some amount of logic there to handle dispatcher vs extension receivers.

Reply:
Thanks! I will definitely give it a look